src/contracts/aludel.ts

```diff
@@ -52,6 +56,139 @@ export async function getNetworkStats(signer: any) {
+    console.log("Wei rewards before multiplier", weiRewards.toString())
+
+    if (stakeDuration <= 5184000) {
+      console.log("less than 60 days")
+      weiRewards = weiRewards
+        .mulUnsafe(FixedNumber.from(stakeDuration))
+        .divUnsafe(FixedNumber.from(rewardScalingPeriod))
+        .mulUnsafe(FixedNumber.from(ceiling - floor)) // 9
+        .divUnsafe(FixedNumber.from(ceiling)); // 10
+    }
+
+    console.log("Wei Rewards after multiplier", weiRewards.toString());
+
+    // Calculate inflation bonuses to mist rewards pools
+    const inflationStart = 1612643118;
+    const inflationPeriodLength = 1209600;
+    const startingSupply = 1000000
+    const inflationRate = 1.01;
+    const inflationToBonusRate = 2;
+    const inflationPeriodsElapsed= Math.floor(
+      (now + daysAdditionalInSecs - inflationStart) / inflationPeriodLength
+    );    
+    let bonusTokensFromInflation = parseEther((( startingSupply * Math.pow(inflationRate, inflationPeriodsElapsed)- startingSupply)/inflationToBonusRate).toString())
+    console.log("Bonuus tokens- ", day, inflationPeriodsElapsed, bonusTokensFromInflation )
+    const futureMistBalance = mistRewardsBalance.add(bonusTokensFromInflation);
+    // Calculate future mist reward pool size
+    const mistRewards = weiRewards
+      .mulUnsafe(FixedNumber.from(futureMistBalance))
+      .divUnsafe(FixedNumber.from(totalUnlockedWeiRewards))
+
+
+    console.log("Mist Rewards", mistRewards.toString());
+    // Convert to string
+    chartData.push({
+      day: `Day ${day}`,
+      "Mist Rewards": parseFloat(mistRewards.toString()),
+      "Ether Rewards": parseFloat(weiRewards.divUnsafe(FixedNumber.from(parseEther('1'))).toString()),
+    });
+  }
+  return chartData;
+}
+
+function getFutureUnlockedRewards(additionlDaysInSeconds: number, rewardBalance: any, sharesOutstanding: any, scheduleShares: any) {
+  const now = Math.floor(Date.now() / 1000); // good
+  console.log("sharesOutstanding", sharesOutstanding);
+  console.log("days in seconds", additionlDaysInSeconds);
+  const scheduleStart = 1613656364; // good
+  const scheduleDuration = 7776000; // good
+  const duration = now + additionlDaysInSeconds - scheduleStart;
+  if (duration > scheduleDuration) {
+    return FixedNumber.from(scheduleShares).mulUnsafe(FixedNumber.from(rewardBalance)).divUnsafe(FixedNumber.from(sharesOutstanding))
+  }
+  // Correct!
+  const sharesLocked = FixedNumber.from(scheduleShares)
+  .subUnsafe(
+    FixedNumber.from(scheduleShares)
+    .mulUnsafe(FixedNumber.from(now - scheduleStart))
+    .divUnsafe(FixedNumber.from(scheduleDuration)))
+  console.log("Shares locked", sharesLocked.toString())
+
+    
+  // const rewardLocked  = sharesLocked
+  //   .mulUnsafe(FixedNumber.from(rewardBalance))
+  //   .divUnsafe(FixedNumber.from(sharesOutstanding))
+
+    const rewardLocked  = sharesLocked
+    .mulUnsafe(FixedNumber.from(rewardBalance))
+    .divUnsafe(FixedNumber.from(scheduleShares)) // ?
```

Comment:
`sharesOutstanding` is what it should be, but I couldn't understand why the `sharesOutstanding` variable was so low